src/cpp/multisolver_interface/environment.cc

```diff
@@ -145,6 +146,12 @@ std::function<int(
   void(XPRS_CC* f_message)(XPRSprob cbprob, void* cbdata, const char* msg, int msglen, int msgtype),
   void* p)>
   XPRSsetcbmessage = nullptr;
+std::function<int(
+  XPRSprob prob,
+  void(XPRS_CC* f_message)(XPRSprob cbprob, void* cbdata, const char* msg, int msglen, int msgtype),
+  void* p,
+  int priority)>
+  XPRSaddcbmessage = nullptr;
```

Comment:
Do we still use this ? (Used for debug in the POC if I remember well)

Reply:
Initially I used it before switching to SolverXpress interface and I forgot to delete it. Good catch!